pkg/transport/generic_transport_controller.go

```diff
@@ -54,7 +57,16 @@ const (
 // NewTransportController returns a new transport controller
 func NewTransportController(ctx context.Context, edgePlacementDecisionInformer edgev2alpha1informers.EdgePlacementDecisionInformer, transport Transport,
 	kubeBindSpaceRelation kbuser.KubeBindSpaceRelation, spaceManagementClient msclient.KubestellarSpaceInterface, spaceProviderNs string,
-	transportSpaceClient dynamic.Interface) *genericTransportController {
+	transportSpaceConfig *rest.Config) (*genericTransportController, error) {
+	transportSpaceDynamicClient, err := dynamic.NewForConfig(transportSpaceConfig)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create dynamic k8s clientset for transport space - %w", err)
+	}
+	wrappedObjectGVK := transport.WrapObjects(make([]*unstructured.Unstructured, 0)).GroupVersionKind() // empty wrapped object to get GVK from it.
```

Comment:
This depends on the fact that transport.WrapObjects returns a wrapped object with an empty list of objects inside it. You need to at least document this.